.rustfmt.toml

```diff
@@ -1,3 +1,4 @@
+edition = "2021"
```

Comment:
This is unnecessary, rustfmt will read the edition from `Cargo.toml`.

Reply:
It depends on your setup. sadly. I added it as a workaround for the `rust.vim` vim plugin, which currently defaults to 2015 when there is a `.rustfmt.toml` without the edition key. See https://github.com/rust-lang/rust.vim/issues/464

Reply:
Can you add a comment above or at the end of the edition line that this is a workaround for that issue? No need to create a separate PR.